### PR TITLE
News-Brazil jumps to No. 2 in worldwide coronavirus cases, behind US (October 27,2020)

### PR DESCRIPTION
**Pull request by Praduman30642**
Hey this Praduman Covid-19 tracker.

### CODE REVIEW

**Comment by mbiesiad** [comment]
Hi @Praduman30642 great idea! But at this moment you're creating a new file here. Could you rename the file for 'News'?

image

If so, with new name (News), could you put your current title "Brazil jumps to No. 2 in worldwide coronavirus cases, behind US" above/down the given code with link? + today' date

Please, rename the file. Put the title inside the field & add the current date.
News -> title "Brazil (...)" -> data & date

The file News should looks like:
# Title_1
Descr._1 / link, etc
date

Thanks! Best wishes

File: Brazil jumps to No. 2 in worldwide coronavirus cases, behind the US [hunk 1]
```diff
@@ -0,0 +1 @@
+<iframe src="https://ourworldindata.org/coronavirus-data-explorer?zoomToSelection=true&country=~BRA&region=World&casesMetric=true&interval=daily&hideControls=true&smoothing=0&pickerMetric=location&pickerSort=asc" loading="lazy" style="width: 100%; height: 600px; border: 0px none;"></iframe>
```
Comment (mbiesiad) [inline Brazil jumps to No. 2 in worldwide coronavirus cases, behind the US:1]: Please add here
Brazil jumps to No. 2 in worldwide coronavirus cases, behind US
2020-10-27
& rename the file to News

**Review by mbiesiad — CHANGES_REQUESTED** [review]

**Comment by Praduman30642** [comment]
Done

**Comment by mbiesiad** [comment]
Thanks. I will fix it, approved 🔥

**Comment by Praduman30642** [comment]
Very very thank you🙏🙏🙏

**Comment by Praduman30642** [comment]
It is not showing approved it's showing under review.
image

**Comment by mbiesiad** [comment]
My pleasure, and thank you too! Ok, I'll take a look at that (in the evening). Let's keep in touch :octocat:

**Comment by mbiesiad** [comment]
By the way, what's about the 1st one? Everything is fine?

**Comment by Praduman30642** [comment]
The first one has same issue.

**Comment by mbiesiad** [comment]
No, it's not. Everything is fine. Next time, please be sure about the Legend:
"Your PR has been accepted by a maintainer and is currently within the review period, which lasts for fourteen days."

You have to wait 14days now. It's not 'Pending' status, it's 'In Review' due to the rules.

Greetings

**Comment by mbiesiad** [comment]
After this period, you will have icon like this

image

Which means: "Accepted - Good job! Your PR has passed the review period and counts toward completing the Hacktoberfest challenge!"

**Comment by Praduman30642** [comment]
Thanks for informing me.😊😊😊😊

**Pull request merged**